doc/source/examples/multiphysics/capillary-wave/capillary-wave.rst

```diff
@@ -0,0 +1,379 @@
+
+.. note::
+  The origin of the plane :math:`\left( (x,y)=(0,0) \right)` is located at the center of the domain.
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/initial-state.svg                                                                             |
+|     :align: center                                                                                                |
+|     :width: 600                                                                                                   |
+|     :name: Initial amplitude of the capillary wave                                                                |
+|                                                                                                                   |
+|     Initial state of the wave                                                                                     |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+Neglecting the gravitational contribution, the phase velocity of the wave may be expressed as:
+
+.. math::
+  c_\sigma = \sqrt{\frac{2\pi\sigma}{\hat{\rho}\lambda_\sigma}}
+
+and the angular frequency simply becomes:
+
+.. math::
+ \omega_\sigma = \sqrt{\frac{\sigma}{\hat{\rho}} \left(\frac{2\pi}{\lambda_\sigma}\right)^3}
+
+Since, the phase fraction (:math:`\phi`) is treated explicitly, the temporal resolution of the capillary wave leads to a Courant-Friedrichs-Lewy (CFL) condition, also known as the *capillary time-step constraint* `[2] <https://doi.org/10.1016/j.jcp.2015.01.021>`_:
+
+.. math::
+  \Delta t_\sigma = \frac{\Delta x}{\sqrt{2} c_\sigma} = \sqrt{\frac{\hat{\rho}}{2\pi\sigma}{{\Delta x}^3}}
+
+with the shortest unambiguously resolved capillary wave having a wavelength of :math:`\lambda_\sigma = 2 \Delta x`.
+
+Therefore, in order to get stable simulation results, :math:`\Delta t < \Delta t_\sigma` should be respected. In this example, different time-steps will be used to explore the stability limit of Lethe's current implementation.
+
+
+--------------
+Parameter File
+--------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+Below, the ``simulation control`` subsection for the case of :math:`\Delta t \approx 0.95\Delta t_\sigma \approx 0.95(3.9 \times 10^{-9})\, \text{s}` is shown. For other cases, the ``time step`` value will change and accordingly the ``output frequency`` will also.
+
+The time integration is handled by a 2nd-order backward differentiation scheme (bdf2) with a constant time-step of :math:`\Delta t=3.7 \times 10^{-9} \, \text{s}`. To assess the stability of the simulation results, the wave is simulated for :math:`t_{\text{end}} = 50\omega_\sigma \approx 4.5 \times 10^{-5} \, \text{s}`.
```

Comment:
Does the omega_sigma have units of time even if it's a frequency? If not it may need fixing

Reply:
Good catch! It's indeed 1/omega_sigma!